src/components/ContactGroup.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+ContactGroup.propTypes = {
+  name: PropTypes.string.isRequired,
+  content: PropTypes.string.isRequired,
+}
+
+export default function ContactGroup({ name, content }) {
+  return (
+    <Wrapper>
+      <Title>{name}</Title>
+      <Value>{content}</Value>
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.li`
```

Comment:
I'm not shure but i think a li needs a ul parent element around.

Reply:
There is a ul as a parent element in the `ContactsPage.js`